Vostok.Metrics.System/Host/NativeHostMetricsCollector_Linux.cs

```diff
@@ -145,6 +180,48 @@ private PerformanceInfo ReadPerformanceInfo()
             return result;
         }
 
+        private NetworkUsage ReadNetworkUsageInfo()
+        {
+            var result = new NetworkUsage();
+
+            try
+            {
+                var totalReceivedBytes = 0L;
+                var totalSentBytes = 0L;
+                var countedInterfaces = new HashSet<string>();
+
+                // NOTE: Skip first 2 lines because they contain format info. See https://man7.org/linux/man-pages/man5/proc.5.html for details.
+                // NOTE: We don't need info from non ethernet interfaces.
+                foreach (var line in networkUsageReader.ReadLines().Skip(2))
+                {
+                    if (FileParser.TrySplitLine(line, 17, out var parts) &&
+                        parts[0].StartsWith("eth") &&
+                        long.TryParse(parts[1], out var receivedBytes) &&
+                        long.TryParse(parts[9], out var sentBytes))
+                    {
+                        countedInterfaces.Add(parts[0].TrimEnd(':'));
+                        totalReceivedBytes += receivedBytes;
+                        totalSentBytes += sentBytes;
+                    }
+                }
+
+                result.ReceivedBytes = totalReceivedBytes;
+                result.SentBytes = totalSentBytes;
+
+                // NOTE: See https://www.kernel.org/doc/Documentation/ABI/testing/sysfs-class-net for details.
+                var networkMaxMBitsPerSecond = countedInterfaces.Sum(
+                    @interface => int.Parse(File.ReadAllText($"/sys/class/net/{@interface}/speed")));
```

Comment:
If it's not possible to calculate max speed, ALL network metrics will be equal to zero.

I made it on purpose:

1. I could omit it, but then I could get amiss results like 150% utilization. (The value is clamped by 0 and 100, but the whole point of this metric is not clear in this case.)
2. It's easier to notice that network metrics are absent (This library couldn't grasp network info in this case.)
3. In the docs we can read that all ethernet interfaces implement needed function so it won't be a problem for our hosts.